### PR TITLE
Add hpa handler for deployment 

### PR DESCRIPTION
**Pull request by zehuaiWANG**
refer to #3867

### CODE REVIEW

**Comment by zehuaiWANG** [comment]
I can help to add the front-end part later if nobody take it.

File: src/app/backend/handler/apihandler.go [hunk 1]
```diff
@@ -321,11 +321,18 @@ func CreateHTTPAPIHandler(iManager integration.IntegrationManager, cManager clie
...
+	apiV1Ws.Route(
+		apiV1Ws.GET("/horizontalpodautoscaler/{kind}/{namespace}/{name}").
+			To(apiHandler.handleGetHorizontalPodAutoscalerListForResource).
+			Writes(horizontalpodautoscaler.HorizontalPodAutoscalerList{}))
 	apiV1Ws.Route(
 		apiV1Ws.GET("/horizontalpodautoscaler/{namespace}/{horizontalpodautoscaler}").
 			To(apiHandler.handleGetHorizontalPodAutoscalerDetail).
 			Writes(horizontalpodautoscaler.HorizontalPodAutoscalerDetail{}))
-
+	apiV1Ws.Route(
```
Comment (floreks) [inline src/app/backend/handler/apihandler.go]: It's the same as above?

Comment (zehuaiWANG) [reply src/app/backend/handler/apihandler.go]: yes, it's the same, I have removed it. Thank you~ And I will help to add the front-end part later if nobody take it.

**Pull request merged**